### PR TITLE
Add constraint type for gravity compensation torque

### PR DESCRIPTION
**Pull request by avalenzu**
Allows planning of postures in which the torques required for specific joints to resist gravity remain within some bounds.

### CODE REVIEW

**Comment by avalenzu** [comment]
Closing and re-opening to bump build servers

**Comment by avalenzu** [comment]
The above failure on drake006 looks like a bug in `solveLCP` (probably same as #932).

File: systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #include "RigidBodyManipulator.h"
 #include "RigidBodyConstraint.h"
 #include "drakeUtil.h"
+#include "controlUtil.h"
```
Comment (RussTedrake) [inline systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp]: what exactly is it that forced the new controlUtil dependence?

Comment (rdeits) [inline systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp]: when I did the controller reorg, I created a `sizecheck()` and put it in `controlUtil`. We can certainly move it elsewhere

Comment (RussTedrake) [inline systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp]: sounds like it should probably go into drakeUtil.

Comment (avalenzu) [inline systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp]: Done. Thanks, @rdeits!

**Comment by RussTedrake** [comment]
@psiorx - have you seen this one (the failing test above): http://kobol.csail.mit.edu/cd/testDetails.php?test=1764441&build=12217

**Comment by psiorx** [comment]
I can't login to CDash (not registered?) , but it could be #932 ?  It happens when solveLCP tries to use the stale cached data from solveLCPmex (fixed).

**Comment by psiorx** [comment]
Just logged into the build server and confirmed that it was #932 . It's fixed in master.

**Pull request merged**